docs/content/author-bundles.md

```diff
@@ -67,11 +70,8 @@ maintainers:
 Mixins are adapters between the Porter and an existing tool or system. They know how to talk to Porter to include everything
 they need to run, such as a CLI or config files, and how to execute their steps in the Porter manifest.
 
-Anyone can [create a mixin](/mixin-dev-guide/), here's a list of the mixins that are installed with Porter by default:
-
-* exec - run shell scripts and commands
-* helm - use the helm cli
-* azure - provision services on the Azure cloud
+There are [many mixins](/mixins/) created both by the Porter Authors and the community.
```

Comment:
nit: Porter authors are part of the community. Are some mixins considered "official" Porter author maintained? or are they all community driven?

Reply:
Some mixins are maintained by the Porter project, i.e. they live in our github org (getporter). Others are maintained by a community member on their own and we aren't involved. I wasn't sure how to say that well.

Reply:
does it need to be separate? because it's possible that mixins would be adopted to the official organization?